### ISSUE TITLE
add alt text for `CardComponent` images

### ISSUE THREAD

**Issue by ix0rai** [umwics/WICS-website-vue#148]
currently, images in the main content of the site (ex. the homepage) have alt text, but anything stored in a `CardComponent` does not have alt text or other identifying attributes. this will require adding a new property to `CardComponent` for alt text!
